src/v2/stack_aggregator.py

```diff
@@ -13,9 +13,11 @@
 
 from typing import Dict, List, Tuple, Set
 from f8a_utils.gh_utils import GithubUtils
+from f8a_utils.ingestion_utils import unknown_package_flow, \
+    Package as P
```

Comment:
Why the Package is aliased to P? it hinders readability.Please keep as it is.

Also use () to have multiline imports.
```suggestion
from f8a_utils.ingestion_utils import (unknown_package_flow, 
    Package)
```

Reply:
Here the name is conflicting with the model name. 
https://github.com/fabric8-analytics/f8a-server-backbone/blob/1367cf92f4a31e4cccb8c6e4074c1e3d489cf53b/src/v2/stack_aggregator.py#L24

Reply:
In that case use fully qualified type.

```
import f8a_utils.ingestion_utils

ingestion_utils.Package(name=,..version=..)
```